packages/mds-types/index.ts

```diff
@@ -149,6 +149,7 @@ export type UUID = string
 export type Timestamp = number
 export type Stringify<T> = { [P in keyof T]: string }
 export type Nullable<T> = T | null
+export type NullableProperties<T> = { [P in keyof T]: Nullable<T[P]> }
```

Comment:
I think you want to make sure that `T` is an object (not an array or primitive) so maybe `<T extends {}>`